.pre-commit-config.yaml

```diff
@@ -0,0 +1,38 @@
+# Auto formats python code
+repos:
+
+-    repo: https://github.com/psf/black.git
+     rev: 23.1a1
+     hooks: 
+     - id: black
+
+
+-    repo: https://github.com/pre-commit/pre-commit-hooks
```

Comment:
Add another comment to this repo to match the Black one?

Reply:
Not really sure what to give as comment for this one since there a lot going on

Reply:
Just say "Runs pre-commit hooks for code formatting" or something like that